### ISSUE TITLE
Notifications bell icon fails contrast checks when no notifications available

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1001]
The inactive state of the notifications component fails WCAG colour contrast requirements.

https://pulsar.docs.jadu.net/components/notifications

![Screenshot 2019-07-08 at 11 33 44](https://user-images.githubusercontent.com/18653/60803958-46589700-a174-11e9-928e-92d6f70634f9.png)
